content/doc/book/managing/casc.adoc

```diff
@@ -70,7 +70,7 @@ and *Manage Jenkins >> Configure Nodes and Clouds* screens.
 * `tool` section defines build tools that can be set on the
 *Manage Jenkins >> Global Tool Configuration* screen.
 
-* *unclassified* section defines all other configurations,
+* 'unclassified' section defines all other configurations,
```

Comment:
```suggestion
* `unclassified` section defines all other configurations,
```
The other keywords are wrapped under ["backticks" (`` ` ``)](https://en.wikipedia.org/wiki/Backtick)  , which is a different character than the simple quotes (`'`).

Backticks, when using Asciidoctor language, means "formating the wrapped text as monospaced": https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#text-formatting

Reply:
Thanks!